tt_metal/hw/inc/dataflow_api.h

```diff
@@ -821,7 +827,7 @@ struct InterleavedAddrGenFast {
     }
 
     FORCE_INLINE
-    void noc_async_read_tile(const uint32_t id, uint32_t dest_addr, const uint32_t offset = 0) const {
+    void noc_async_read_tile(const uint32_t id, uint32_t dest_addr, const uint32_t offset = 0, uint32_t vc = 0) const {
```

Comment:
Why not keep consistent with the other parameters and do `const uint32_t vc = 0`?

Reply:
vc is not constant, we need to change it for different channels

Reply:
ok